package.json

```diff
@@ -36,27 +36,116 @@
     "configuration": {
       "properties": {
         "purple-yolk.cabal.formatter.command": {
-          "default": "cabal-fmt --no-cabal-file --no-tabular",
+          "default": "",
```

Comment:
This is a noteworthy change. If you previously customized the Cabal formatter command, you'll now have to set the mode to "custom" for that command to be used.